src/components/UI/Text/Textarea.vue

```diff
@@ -36,26 +36,30 @@
       >
         <inline-svg :src="startIconObj"></inline-svg>
       </div>
-
       <!-- input text area -->
-      <textarea
-        class="p-2 border placeholder-blueGray-300 text-blueGray-600 bg-white disabled:bg-gray-200 rounded text-md border-blueGray-300 focus:outline-none focus:ring focus:border-transparent focus:shadow-outline w-full border-gray-200 disabled:cursor-not-allowed"
+
+      <div
+        contenteditable="true"
+        class="pt-7 h-30 textbox pl-0 border placeholder-blueGray-300 text-blueGray-600 bg-white disabled:bg-gray-200 rounded text-md border-blueGray-300 focus:outline-none focus:ring focus:border-transparent focus:shadow-outline w-full border-gray-200 disabled:cursor-not-allowed"
         :class="[inputAreaClass, boxStyling]"
         :disabled="isDisabled"
+        ref="editor"
         :placeholder="placeholder"
-        v-model="localValue"
+        @input="inputChange"
         name="placeholder"
         autocomplete="off"
-        @input="inputChange"
         @keypress="keyPress"
         @keydown="keyDown"
         data-test="input"
-      />
+      ></div>
     </div>
   </div>
 </template>
 
 <script>
+import Quill from "quill";
+import "quill/dist/quill.snow.css";
```

Comment:
Add comment saying that importing css file is necessary and icons wont appear properly without it

Reply:
okay